theta: [-1.570796327, 0, 3.141592654]
alpha: [0, 0, 0]
d: [0, 0, 0]
a: [0.083, 0.064, 0.05]
origin:
  left_arm: [0, -0.1583424671, -0.07349, 0, 3.14, 0]
  right_arm: [0, 0.1583424671, -0.07349, 0, 3.14, 3.14]